libheif/heif.cc

```diff
@@ -458,6 +458,44 @@ void heif_context_debug_dump_boxes_to_file(struct heif_context* ctx, int fd)
   (void) written;
 }
 
+
+//Created by Devon Sookhoo
+LIBHEIF_API
+void heif_add_box_example(char* input_filename) {
+
+  //CREATE CONTEXT 
+  std::shared_ptr<heif_context> ctx(heif_context_alloc(), [](heif_context* c) { heif_context_free(c); });
+  if (!ctx) {
+    fprintf(stderr, "Could not create context object\n");
+    exit(1);
+  }
+
+  //OPEN IMAGE
+  struct heif_error err = heif_context_read_from_file(ctx.get(), input_filename, nullptr);
+  if (err.code != 0) {
+    std::cerr << "Could not read HEIF/AVIF file: " << err.message << "\n";
+    exit(1);
+  }
+
+  //GET HEIF FILE
```

Comment:
These comments aren't adding much.  The code itself should read like prose.  We shouldn't need a bunch of comments to describe what should easily be readable in code.